### ISSUE TITLE
newline after opening brace

### ISSUE THREAD

**Issue by justechn** [squizlabs/PHP_CodeSniffer#1521]
I have this line of code
```
function ($v, $k) { return sprintf("%s=%s", $k, $v); },
```

and I want to setup a ruleset that will require a newline after the opening curly brace. However I am having trouble getting it to work. 

I am currently using the PSR2 ruleset which has a rule `<rule ref="Squiz.ControlStructures.ControlSignature"/>` which I think is supposed to contain the [rule I am looking for](https://github.com/squizlabs/PHP_CodeSniffer/blob/b95ff2c3b122a3ee4b57d149a57d2afce65522c3/src/Standards/Squiz/Sniffs/ControlStructures/ControlSignatureSniff.php#L148), but it does not throw an error.

Am I missing anyting?

**Comment by gsherwood** [comment]
When I run the PSR2 standard over that code, I get these two errors:
```
FILE: temp.php
-------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------
 2 | ERROR | [x] Opening brace must be the last content on the line (Squiz.Functions.MultiLineFunctionDeclaration.ContentAfterBrace)
 2 | ERROR | [x] Closing brace must be on a line by itself (Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore)
-------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------------------------------
```
If I use the diff report to see the suggested fixes, I get this:
```diff
--- temp.php
+++ PHP_CodeSniffer
@@ -1,2 +1,4 @@
 <?php
-function ($v, $k) { return sprintf("%s=%s", $k, $v); }
+function ($v, $k) {
+    return sprintf("%s=%s", $k, $v);
+}
```
So it all looks fine to me, except you're maybe looking at the wrong sniff. The one doing this specific function check is `Squiz.Functions.MultiLineFunctionDeclaration`.

**Comment by justechn** [comment]
thanks, I was looking at the wrong sniff
